docs/website/theory/theoryGrid.md

```diff
@@ -81,22 +97,40 @@ supposed to be used together with the grid implementation `SimpleGrid`.
 It is based on three template parameters `griddim`(dimension of the grid), `cogriddim`(???) and 
 `wdim`(dimension of the world).
 
-ToDo: How are the private quantities (e.g. entitiesFathers) constructed?
+@ Alex: Could you explain the implementation details here? 

```

Comment:
Wir haben uns doch darauf geeinigt mit Malte, dass man Implementierungsdetails nicht in die Dokzu packt sondern die nur im Code dokumentiert. D.h. hier müssen wir nur das Interface besprechen